OpenDreamClient/Interface/Controls/ControlWindow.cs

```diff
@@ -244,7 +246,7 @@ private void CreateMenu(MenuDescriptor menuDescriptor)
         private MenuBar.MenuEntry CreateMenuItem(string name, string command, bool isCheckable)
         {
             if (name.StartsWith("&"))
-                name = name.Substring(1); //TODO: First character in name becomes a selection shortcut
+                name = name[1..]; //TODO: First character in name becomes a selection shortcut
```

Comment:
Is this still needed?

Reply:
Yes. The `menu.Title` change above doesn't affect the name of the items within the menu.